### PR TITLE
move licenses file & include with builds

### PR DESCRIPTION
**Pull request by jeffvg**
Fixes # .

#### Changes proposed in this Pull Request:
- Move license file from root to each dist folder
- Add additional file tests
- Add file name to files list in package.json

#### To Test:
- 
open branch and remove the licenses.json from both the branding and UI folders.
Run yarn precommit
Licenses.json should be created and place back in Branding/UI folders
Test should reflect that the licenses are found

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
         "build:colors-branding": "cd branding && yarn build && cd ..",
         "build:all": "yarn build:colors && yarn build:colors-branding",
         "prettier": "prettier \"**/**.{ts,tsx,js,jsx,json,css,scss,html,md}\" --write",
-        "generate:licenses": "npm-license-crawler -onlyDirectDependencies -json LICENSES.json"
+        "generate:licenses": "cd ./ui && npm-license-crawler -onlyDirectDependencies -json LICENSES.json && cd .. & cd ./branding && npm-license-crawler -onlyDirectDependencies -json LICENSES.json"
```
Comment (daileytj) [inline package.json:12]: Seems to work fine without the second & for some reason, but we should probably do this
```suggestion
        "generate:licenses": "cd ./ui && npm-license-crawler -onlyDirectDependencies -json LICENSES.json && cd .. && cd ./branding && npm-license-crawler -onlyDirectDependencies -json LICENSES.json"
```

**Comment by daileytj** [comment]
What's actually supposed to happen here? 

I'm assuming that when this package is published that it should have the populated licenses file shipped with it?
If so, there are a few issues here:
- There's no documentation or instruction to ensure that this is updated/supported long-term.
- On a fresh clone the nested packages will not have ```node_modules``` installed, resulting in an unpopulated ```LICENSES.json``` file on a fresh run of ```generate:licenses```, so that should probably be built into the script unless this will only be used by circleci.
- Once library ```node_modules``` are installed, the licenses file is being copied to the src files, but when the package is built the ```LICENSES.json``` file is not included in the dist folder

**Comment by daileytj** [comment]
Can you update this PR to share testing instructions as well as be sure to add test details to anything that doesn't seem obvious.

**Comment by jeffvg** [comment]
on a fresh clone the licenses files should exists. To update we just yarn install and run yarn generate:licenses.  We should include the generate licenses in the build. Should we just add a "pre-commit" to the scripts?

to test - delete the licenses.json files from both directories. yarn && yarn generate:licenses

**Comment by jeffvg** [comment]
in the end we simply want the licenses json file to live with everything else in the package-
image

**Comment by jeffvg** [comment]
see Joes comments in this PR - 
https://github.com/brightlayer-ui/angular-component-library/pull/412

**Comment by jeffvg** [comment]
forgot to mention the files are referenced in each package.json for UI and Branding...

image

**Pull request merged**